app/controllers/public_health_controller.rb

```diff
@@ -278,6 +290,100 @@ def linelist_specific_fields(workflow, tab)
     %i[jurisdiction assigned_user end_of_monitoring risk_level monitoring_plan latest_report report_eligibility]
   end
 
+  def advanced_filter(patients, filters)
+    filters.each do |filter|
+      case filter[:filterOption]['name']
+      when 'sent-today'
+        patients = patients.where("last_assessment_reminder_sent #{filter[:value] ? '>' : '<'} ?", DateTime.now.beginning_of_day)
```

Comment:
Yeah, it's a little tricky in this case because we don't have a quick "patient" to grab to figure out the right timezone. I thought maybe of trying to rephrase the option to be something like, "the last 24 hours" or something, which eliminates the problem, but not sure about that. Thoughts?